### ISSUE TITLE
Seeding the database / .dump file clarifications

### ISSUE THREAD

**Issue by Utopiah** [icosa-foundation/icosa-gallery#73]
Hi,

I'm looking for a way to integrate with a library of low poly models offline (for privacy concerns and to give reliable workshops) to facilitate in VR content creation. For now I have manually been downloading Google Blocks made models from the Poly unofficial successor, poly.pizza

Ideally I would rely on an entirely open-source solution. I do not want to scrap poly.pizza nor do I want to implement a gallery itself. I believe this project could thus help a lot.

Beyond the gallery itself though I'm interested in content, i.e free to use low poly 3D models. I believe the ReadMe section mentions seeding the DB and using .dump files. I have not seen examples though, nor link to such files.

Could you please clarify?

**Comment by Utopiah** [comment]
FWIW https://wiki.archiveteam.org/index.php/Google_Poly leads to 50Gb dumps e.g `googlepoly_20210630022356_20297ef9.megawarc.warc.gz | 30-Jun-2021 03:51 | 49.3G` in https://archive.org/download/archiveteam_googlepoly_20210630022356_20297ef9 so wondering if that's usable?

**Comment by andybak** [comment]
Yeah - the warc files were invaluable and I've been using them locally to extract the missing files I've needed.

We will have a public dataset available soon - we used a similar starting point to https://polygone.art/ but we've filled in a lot of missing metadata and links to alternate file versions/formats. There are also an additional 20,000 models but I'm not sure at this stage why they were omitted from polygone.

We'll release the metadata as JSON. The files are currently hosted on our Backblaze B2 (core GLTF and thumbnals) with direct links to the Internet Archive for everything else.

DM me on your social platform of choice and I can give you early access. My username is the same on most platforms.

**Comment by Utopiah** [comment]
How? :P

**Comment by andybak** [comment]
Oooops. Sorry. Reopening. I thought I'd answered your question and therefore resolved the issue.

**Comment by Utopiah** [comment]
Well you definitely did help me and thanks to you I found a way to move forward. Yet I believe at least the README should either reflect that or be amended so that me (in case I forget) or others can find a way too.

**Comment by jamesrampton** [comment]
@Utopiah Indeed, I'll be overhauling the README soon so it's a lot more useful and assumes a lot less about the reader.
